consensus/state_exchain.go

```diff
@@ -32,16 +33,19 @@ type coreData struct {
 }
 
 type consensusTrack struct {
+	lock sync.Locker
 	coreTrack   map[int64]*coreData
 	trackSwitch bool
 	l           log.Logger
 }
 
 func newConsensusTrack(trackSwitch bool) *consensusTrack {
-	return &consensusTrack{make(map[int64]*coreData), trackSwitch, nil}
+	return &consensusTrack{&sync.Mutex{}, make(map[int64]*coreData), trackSwitch, nil}
```

Comment:
why add lock here?  this is not a good idea that add lock into a log trace just for fix unit test